region/norfair/crocomire/Grapple Tutorial Room 2.json

```diff
@@ -175,6 +254,35 @@
       "name": "Base",
       "requires": []
     },
+    {
+      "link": [2, 1],
+      "name": "Come In Shinecharging, Leave With Spark",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 0,
+          "openEnd": 1
+        }
+      },
+      "requires": [
+        "canShinechargeMovementTricky",
+        {"shinespark": {"frames": 5, "excessFrames": 0}}
```

Comment:
Reaching the door with frames left isn't hard.

My best was 20 frames remaining, if I have grapple or wave for the door/

Reply:
Its related to your entry speed. I see 17 frames with minimal entry speed and 27 running full speed into the room.